### PR TITLE
Use github actions as test CI

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q             | A
| ------------- | ---
| Branch?       | "master
| Bug fix?      | no
| New feature?  | -
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

See https://github.com/alexander-schranz/routing-bundle/pull/1

Requires:

 - [x] https://github.com/symfony-cmf/Testing/pull/208

### CODE REVIEW

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+            - name: Require Specific Symfony Version
+              if: ${{ matrix.symfony-version }}
+              run: composer require --no-update symfony/symfony:${{ matrix.symfony-version }}
+
+            - name: Install dependencies with Composer
+              uses: ramsey/composer-install@v1
+              with:
+                  dependency-versions: ${{ matrix.dependencies }}
+                  composer-options: --prefer-dist --no-suggest
```
Comment (dbu) [inline .github/workflows/test-application.yaml:61]:
```suggestion
                  composer-options: --prefer-dist
```

File: .github/workflows/test-application.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  tools: 'composer:v2'
+
+            - name: Require Specific Symfony Version
+              if: ${{ matrix.symfony-version }}
+              run: composer require --no-update symfony/symfony:${{ matrix.symfony-version }}
```
Comment (dbu) [inline .github/workflows/test-application.yaml:55]: i would prefer to keep using flex for managing this. requiring symfony/symfony will hide it if we missed to declare the dependency on a specific symfony component.

**Review by dbu — COMMENTED** [review]
thanks for this one too!

the php 7.2 build seems to be missing the mysql driver

i would prefer to keep using symfony flex (see the deleted travis file how we worked with that) to lock symfony versions.

File: .github/workflows/test-application.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
+
+jobs:
+    test:
+        name: 'PHP ${{ matrix.php-version }}, Symfony ${{ matrix.symfony-version }}'
```
Comment (dbu) [inline .github/workflows/test-application.yaml:14]:
```suggestion
        name: 'PHP ${{ matrix.php-version }}, Symfony ${{ matrix.symfony-version }} ${{ matrix.dependencies}}'
```

**Review by dbu — COMMENTED** [review]
thanks. php 7.2 is missing the pdo driver (its sqlite not mysql, actually)

**Comment by alexander-schranz** [comment]
@dbu Thx did fix it!

**Review by dbu — APPROVED** [review]
cheers!

**Pull request merged**